modules/listInput.js

```diff
@@ -0,0 +1,14 @@
+const listView = document.querySelector('.list');
+const listInputLine = (task) => {
+  const listElement = document.createElement('li');
+  listElement.classList.add('task-item');
+  listElement.innerHTML = `
+    <input type="checkbox" class="task-check">
+    <input type="text" name="${task.description}" class="task-description" value="${task.description}">   
+    <i class="bi bi-three-dots-vertical"></i>
+    <i class="bi bi-trash hide"></i>
+    `;
```

Comment:
I would suggest to try avoiding using `inner.html` because it's very taxing for the browser and because it has some security issues. Please consider using the `inserAdjacentHtml` method over it.

Reply:
Thanks for the suggestion.